Anatomy_of_an_ASDF_file.ipynb

```diff
@@ -154,7 +155,10 @@
    "outputs": [],
    "source": [
     "# Print as string, but requires conversion of text part\n",
-    "with open('tut_with_data.asdf','rb') as taf2: print(taf2.read()[:852].decode(\"utf-8\"))"
+    "with open('tut_with_data.asdf','rb') as taf2:\n",
+    "    fbytes = taf2.read()\n",
+    "    end = fbytes.find(b'...')\n",
```

Comment:
This is only important for modeling best practice, but matching on "..." alone isn't generally safe, since strings in the YAML are permitted to have that sequence embedded in them.  There's a regular expression `asdf.constants.YAML_END_MARKER_REGEX` that we can search for to find the end of the YAML region.

Reply:
Certainly true, but since I construct the example, I'll be sure to keep such strings out :-). I don't think we need to go that far to make this kevlar level.